nautobot/extras/jobs.py

```diff
@@ -1153,9 +1154,11 @@ def run_job(self, job_class_path, *args, **kwargs):
     job.request = self.request
     try:
         job.before_start(self.request.id, args, kwargs)
+        publish_event(topic="nautobot.events.jobs.started", payload=kwargs)
```

Comment:
Probably all of these should just be `nautobot.job.*` topics - please also make sure they're documented in `nautobot/docs/user-guide/platform-functionality/events.md`.

Reply:
Also not sure about the payload - I'd think at the very least it should include the `job_class_path` and the `job_result` PK, not sure about the actual execution kwargs as we need to consider `has_sensitive_variables` in that case?